modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -292,18 +292,17 @@ bool Visualizer::adjust_angles(
 
 
 bool Visualizer::SetDirectory(const std::string &path) {
-  int is_success = 1;
-  std::string command;
-  command = "mkdir -p " + path;
-  is_success = system(command.c_str());
+  std::string command = "mkdir -p " + path;
+  int is_success = system(command.c_str());
+  if (!is_success) {
+    return false;
+  }
+
   command = "rm " + path + "/*.jpg";
   is_success = system(command.c_str());
   path_ = path;
-  if (is_success > 0) {
-    return true;
-  } else {
-    return false;
-  }
+
+  return !is_success ? true : false;
```

Comment:
Plz use utils in cyber/common/file.h
You may need EnsureDirectory() and RemoveAllFiles()

Reply:
Thanks for your comments! Those combination looks good to me in this case. However, RemoveAllFiles() might cause issues if some other kind of files resides in the desired directory. I'd like to close this PR for now, will back to get a deep look at it once another high priority task is completed.